src/engine/renderer/tr_decals.cpp

```diff
@@ -190,7 +190,6 @@ void RE_ProjectDecal( qhandle_t hShader, int numPoints, vec3_t *points, vec4_t p
 
 	/* basic setup */
 	temp.shader = R_GetShaderByHandle( hShader );  /* debug code */
-	temp.numPlanes = temp.shader->entityMergable;
 	temp.color = color;
 	temp.numPlanes = numPoints + 2;
```

Comment:
How do you know there was no bug and `temp.shader->entityMergable` had to be used somehow?

Reply:
If you don't know, please add a comment like this one (feel free to reword):

```c++
	/* Note: the code was storing this value before overwriting it the next line.
	If something had to be done with this value, there may be bugs around there:
	temp.numPlanes = temp.shader->entityMergable;
	*/
	temp.numPlanes = numPoints + 2;
```

This way if someone reports a bug with decals, it would help.

Reply:
Done, thanks!